tests/unit/amo/components/TestAddon.js

```diff
@@ -1128,19 +1133,19 @@ describe(__filename, () => {
       ).addon;
       root.setProps({ addon: addonFromState });
 
-      sinon.assert.calledWith(fakeDispatch, fetchOtherAddonsByAuthors({
+      sinon.assert.calledWith(fakeDispatch, fetchAddonsByAuthors({
         addonType: newAddon.type,
         authors: newAddon.authors.map((author) => author.username),
         errorHandlerId: createStubErrorHandler().id,
-        slug: newAddon.slug,
+        excludeAddonBySlug: newAddon.slug,
       }));
     });
 
     it('puts "add-ons by author" in main content if type is theme', () => {
       const addon = fakeTheme;
       const { store } = dispatchAddonData({
         addon,
-        addonsByAuthors: [{ ...fakeTheme, slug: 'another-slug' }],
+        addonsByAuthors: defaultAddonsByAuthors,
```

Comment:
The old test was using `...fakeTheme` but the helper uses `...fakeAddon`.

I don't think `defaultAddonsByAuthors` is helpful because it obscures the intent of `[{ ...fakeTheme, slug: 'another-slug' }]` which was to make it clear that a *different* slug would be returned by the API.

Reply:
Shoot, yes, my bad.